docs/clients/grpc/persistent-subscriptions.md

```diff
@@ -19,7 +38,7 @@ Once you have created a subscription group, clients can connect to that subscrip
 
 The most important parameter to pass when connecting is the buffer size. This represents how many outstanding messages the server should allow this client. If this number is too small, your subscription will spend much of its time idle as it waits for an acknowledgment to come back from the client. If it's too big, you waste resources and can start causing time out messages depending on the speed of your processing.
 
-@[code{subscribe-to-persistent-subscription-to-stream}](../dotnet/21.2/samples/persistent-subscriptions/Program.cs)
+@[code{subscribe-to-persistent-subscription-to-stream}](../dotnet/21.6/samples/persistent-subscriptions/Program.cs)
```

Comment:
@oskardudycz do we have sample for this in the other langs ?

Reply:
Also, let me check if it is replaceable with an alias.

Reply:
I forgot to update that. It's done with an alias.